lib/src/dynamics/contacts/contact_register.dart

```diff
@@ -3,6 +3,6 @@ import 'dart:collection';
 import '../../../forge2d.dart';
 
 class ContactRegister {
```

Comment:
I don't think this file is used anymore, you could remove it completely if you want.